db/column_family.cc

```diff
@@ -675,6 +675,19 @@ ColumnFamilyData::~ColumnFamilyData() {
           id_, name_.c_str());
     }
   }
+
+  if (data_dirs_.size()) {  // Explicitly close data directories
+    Status s = Status::OK();
+    for (auto& data_dir_ptr : data_dirs_) {
+      if (data_dir_ptr) {
+        s = data_dir_ptr->Close(IOOptions(), nullptr);
+        if (!s.ok()) {
+          ROCKS_LOG_WARN(ioptions_.logger, "Ignoring error %s",
+                         s.ToString().c_str());
+        }
```

Comment:
I'd go with `s.PermitUncheckedError()` because this hopefully isn't the final way of reporting errors. We could have `Status ColumnFamilyData::Close()` or `Status ColumnFamilyData::CloseDirectories()` in the future. `PermitUncheckedError()` is nice because it allows us to easily search for the places error handling is incomplete.

It's fine either way. You could also do both or add a TODO to make this unfinished error handling searchable.